v1/telemetry.js

```diff
@@ -1358,6 +1358,27 @@ Histogram.prototype.count = function Histogram_count() {
 };
 
 /**
+ * Returns the sum of all measurements in this histogram.
+ *
+ *    // Print sum
+ *    if (histogram.kind() == 'count' || histogram.kind() == 'linear' ||
+ *        histogram.kind() == 'exponential') {
+ *      console.log("sum: " + histogram.sum());
+ *    }
+ *
+ * **Remark**, this method is **only supported** for `'count'`, `'linear'`, and
+ * `'exponential'` histograms. See `Histogram.kind()` to see what kind of
+ * histogram you have. Invoking this method on any other kind of histogram will
+ * throw an exception.
+ */
+Histogram.prototype.sum = function Histogram_sum() {
```

Comment:
Other uses of `_aggregate(DataOffsets.SUM, this)` throughout the file should be replaced with this function where possible.